mopidy_musicbox_webclient/static/upload.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+
+<script type="text/javascript" src="vendors/jquery/jquery-1.12.0.min.js"></script>
+<script type="text/javascript" src="vendors/jquery_cookie/jquery.cookie.js"></script>
+<script type="text/javascript" src="js/custom_scripting.js"></script>
+
+<link rel="stylesheet" type="text/css" href="vendors/jquery_mobile_flat_ui_theme/jquery.mobile.flatui.min.css"/>
+<link rel="icon" type="image/gif" href="images/icons/musicbox32.gif" />
+<link rel="apple-touch-icon" href="images/icons/musicbox57.png" />
+<link rel="apple-touch-icon" sizes="72x72" href="images/icons/musicbox72.png" />
+<link rel="apple-touch-icon" sizes="114x114" href="images/icons/musicbox114.png" />
+<link rel="shortcut icon" type="image/x-icon" href="images/icons/musicbox32.gif" />
+
+<meta name="apple-mobile-web-app-status-bar-style" content="black" />
+<meta name="apple-mobile-web-app-title" content="Upload music" />
+<meta name="mobile-web-app-capable" content="yes" />
+<meta name="HandheldFriendly" content="True" />
+<meta name="MobileOptimized" content="320" />
+<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1, user-scalable=no" />
+<meta name="apple-mobile-web-app-capable" content="yes" />
+<meta http-equiv="imagetoolbar" content="no" />
+<meta content="true" name="MSSmartTagsPreventParsing" />
+
+<meta http-equiv="cleartype" content="on" />
+
+<script src="vendors/jquery_mobile/jquery.mobile-1.3.2.min.js"></script>
+<meta name="description" content="MusicBox" />
+<meta name="author" content="Federico Barresi" />
+<meta name="copyright" content="(c) 2017 Federico Barresi" />
+<link rel="stylesheet" type="text/css" href="css/webclient.css" />
+<link rel="stylesheet" type="text/css" href="vendors/font_awesome/css/font-awesome.min.css" />
+<title>Upload Music</title>
+</head>
+<body>
+  <div data-role="page" id="page" class="ui-responsive-panel" data-theme="c" data-title="Upload music">
+
+    <div data-role="header" data-tap-toggle="false" id="header" data-position="fixed" class="header-breakpoint headerbtn">
+        <a id="headermenubtn" href="index.html"><i class="fa fa-chevron-left"></i></a>
+        <h1 id="contentHeadline">Upload Music</h1>
+    </div>
+
+<div data-role="content" class="pane" id="uploadpane">
+  <div class="ui-grid-a pl-breakpoint">
+{% if can_upload %}
+      <h4>Uploading into: {{upload_path}}</h4>
+      <div class="ui-block-a" style="padding: 5px">
+        <form enctype="multipart/form-data" method="post">
+          <p>
+            Subpath: <input type="text" name="subpath" />
```

Comment:
This might be a little input-error-prone as user's won't be able to see which subfolders already exist from this page, and typos will happen.